kw

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
 
-# Set required variables
-EASY_KERNEL_WORKFLOW=${EASY_KERNEL_WORKFLOW:-"kw"}
-src_script_path=${src_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src"}
-plugins_path=${plugins_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src/plugins"}
-etc_files_path=${etc_files_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/etc"}
-config_files_path=${config_files_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW"}
-kw_dir=${kw_dir:-"$HOME/kw"}
-
-# Export external variables required by kworkflow
-export EASY_KERNEL_WORKFLOW
+# Note: the following line should not be modified and neither the token should
+# be repeated in this file. It will be replaced during installation with the
+# respective installation prefix.
+KW_INSTALL_PREFIX=${KW_INSTALL_PREFIX:-"##KW_INSTALL_PREFIX_TOKEN##"}
+
+# Major install paths
+KW_LIB_DIR=${KW_LIB_DIR:-"$KW_INSTALL_PREFIX/lib/kw/"}
+KW_SHARE_DIR=${KW_SHARE_DIR:-"$KW_INSTALL_PREFIX/share/kw/"}
+KW_DATA_DIR=${KW_DATA_DIR:-"$HOME/.kw/"}
+KW_GLOBAL_CONFIG_DIR=${KW_GLOBAL_CONFIG_DIR:-"/usr/etc/kw/"}
+
+# Derivated or non-major paths
+KW_PLUGINS_DIR=${KW_GLOBAL_CONFIG_DIR:-"$KW_LIB_DIR/plugins/"}
+KW_CACHE_DIR="$HOME/.cache/kw/"
+if [[ ! -d "$KW_CACHE_DIR" ]]; then
+  mkdir -p "$KW_CACHE_DIR"
+fi
```

Comment:
I noticed that most of the test failures happen due to the deletion of the`src_script_path` variable, after a couple of minutes trying to figure out how we can fix it I came with the following idea:
1. Move all of the above variables from kw file, and place them in a separate file (maybe env_variables.sh?).
2. Load the new file here.
3. Make the setup.sh finding the new file.
4. Rework `run_tests` for loading the new file with environment variables, and put the correct path in KW_INSTALL_PREFIX.

An alternative for step (4), could change the `run_test prepare` but I'm afraid that this approach may cause obscure problems during the test execution.

Reply:
Any comment?